app/src/main/java/tech/relaycorp/courier/domain/StoreMessage.kt

```diff
@@ -53,7 +53,7 @@ class StoreMessage
         }
 
         try {
-            cca.validate()
+            cca.validate(null)
```

Comment:
This should be `RecipientAddressType.PUBLIC`: https://github.com/relaycorp/relaynet-courier-android/pull/172/commits/dc534568978a72c9d23772b50d338fb89b266f91#diff-387d88eae5ee3e8d42b022b967cb30feR57

You probably copied it from the PoC branch, where I used `null` in a hurry to get it out of the way with the other stuff I needed to test.